diff --git a/openy_af4_vue_app/src/components/ResultsList.vue b/openy_af4_vue_app/src/components/ResultsList.vue
index ccd49c9c..96e2e741 100644
--- a/openy_af4_vue_app/src/components/ResultsList.vue
+++ b/openy_af4_vue_app/src/components/ResultsList.vue
@@ -14,6 +14,11 @@
           </span>
         </div>

+        <div class="row">
+            <span v-if="item.program_subcategory" class="program col-sm-9">
+              <span class="program-label">{{ item.program_subcategory }}</span>
+            </span>
+        </div>
         <div class="ages-spots">
           <span v-if="item.ages || (selectedAges.length && !legacyMode)" class="ages">
             <span class="age-label">{{ 'Ages' | t }}:</span>
@@ -106,7 +111,11 @@
             </template>
           </span>
         </div>
-
+        <div class="row">
+            <span v-if="item.program_subcategory" class="program col-sm-9">
+              <span class="program-label">{{ item.program_subcategory }}</span>
+            </span>
+        </div>
         <div class="row">
           <div class="col-sm-4">
             <div v-if="item.dates" class="item-detail dates">
@@ -243,12 +252,19 @@ export default {
       line-height: 21px;
       color: $af-blue;
       font-weight: bold;
-      margin-bottom: 10px;
-
       @include media-breakpoint-up('lg') {
         font-size: 18px;
         line-height: 27px;
-        margin-bottom: 20px;
+      }
+    }
+
+    .program {
+      margin-top: 5px;
+      margin-bottom: 20px;
+      font-size: 12px;
+      color: $af-blue;
+      @include media-breakpoint-down('md') {
+        margin-bottom: 0;
       }
     }

diff --git a/openy_af4_vue_app/src/components/modals/ActivityDetails.vue b/openy_af4_vue_app/src/components/modals/ActivityDetails.vue
index 645092c5..3b335aac 100644
--- a/openy_af4_vue_app/src/components/modals/ActivityDetails.vue
+++ b/openy_af4_vue_app/src/components/modals/ActivityDetails.vue
@@ -14,6 +14,11 @@
             <div class="left">
               <div class="title">{{ item.name }}</div>
               <div class="description">{{ item.description }}</div>
+              <div v-if="item.program_subcategory" class="row program">
+                <span class="program-link col-sm-9">
+                  {{ item.program_subcategory }}
+                </span>
+              </div>
               <div v-if="item.ages" class="row ages">
                 <div class="col-3 col-xs-3">
                   <strong>{{ 'Ages:' | t }}</strong>
@@ -401,7 +406,8 @@ export default {
     .title,
     .description,
     .ages,
-    .gender {
+    .gender,
+    .program {
       margin-bottom: 10px;
     }

@@ -414,7 +420,8 @@ export default {
     .description,
     .ages,
     .gender,
-    .learn-more {
+    .learn-more,
+    .program {
       font-size: 12px;
       line-height: 18px;
     }
