diff --git a/openy_af4_vue_app/src/main.js b/openy_af4_vue_app/src/main.js
index b4fdcff..d85c387 100644
--- a/openy_af4_vue_app/src/main.js
+++ b/openy_af4_vue_app/src/main.js
@@ -13,6 +13,14 @@ document.addEventListener('openy_activity_finder_event', e => {
       event_label: label,
       value: value
     })
+  } else if (window.dataLayer) {
+    window.dataLayer.push({
+      'event': 'gaGenericEvent',
+      'gaCategory': category,
+      'gaLabel': label,
+      'gaAction': action,
+      'gaValue': value
+    });
   } else if (window.ga) {
     window.ga('send', 'event', category, action, label, value)
   }
